src/main/java/frc/robot/subsystems/shooter/ShooterHardware.java

```diff
@@ -24,61 +24,56 @@ public class RealShooter implements ShooterIO {
     public double pvalue = 0.01;
     private double slewRate = 0;
 
-    public RealShooter()
-    {
-        shooterMotorControllerLow = MotorUtil.createSparkMAX(ShooterConstants.SHOOT_LOW_MOTOR_ID, MotorType.kBrushless, 
-            Constants.NEO_CURRENT_LIMIT, true, true, slewRate);
-        
-        shooterMotorControllerHigh = MotorUtil.createSparkMAX(ShooterConstants.SHOOT_HIGH_MOTOR_ID, MotorType.kBrushless, 
-            Constants.NEO_CURRENT_LIMIT, true, true, slewRate);
+    public ShooterHardware() {
+        shooterMotorControllerLow = new CANSparkMax(ShooterConstants.SHOOT_LOW_MOTOR_ID, MotorType.kBrushless);
+        shooterMotorControllerLow.restoreFactoryDefaults();
+        shooterMotorControllerLow.setSmartCurrentLimit(Constants.NEO_CURRENT_LIMIT);
+        shooterMotorControllerLow.setInverted(true);
+        shooterMotorControllerLow.setIdleMode(CANSparkMax.IdleMode.kBrake);
```

Comment:
was this on brake mode before? I feel like coast would be better for this motor so I'm curious if you based this on the past value